package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/jest-dom": "^5.16.5",
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/226114533-579b7849-14e8-4d9c-8c12-b1e35b915853.png)

- [ ] Kindly complete this message such as **If you like this project then don't forget to give a star :star: on this repository** to make your readme file descriptive enough :blush:

Reply:
Thanks @HAMMAS-SALEEM . I'll update it.